src/Message/SmsMessageResultStatus.php

```diff
@@ -0,0 +1,41 @@
+<?php
```

Comment:
+1000 to separating entire result status from individual recipient report status. I was waiting for you to finish before commenting on this.